jazkarta/shop/browser/controlpanel.py

```diff
@@ -60,7 +60,7 @@ def _fetch_orders(part, key=()):
                 else:
                     href = title = i.get('href', '')
 
-                item_str += '<li><a href="{}">{}</a> x {} @ ${}</li>'.format(
+                item_str += u'<li><a href="{}">{}</a> x {} @ ${}</li>'.format(
```

Comment:
For this to help I think you'll need to use the `u` prefix on all of the literals that are added to `item_str`, i.e. on lines 49 and 66 too.

Reply:
Of course!  I forgot about that in my commit.